### ISSUE TITLE
Feature Request: add directories

### ISSUE THREAD

**Issue by andygrillo** [farminf/S3-webapp#1]
This works great, however it would be very useful if a user could only see an explorer type view with folders and files. This way they can navigate to the folder required, rather than see every file in the main view. cheers!

**Comment by farminf** [comment]
yeah, sure I wanted to do that in fact... I'll do it whenever I find time ;)
Thanks :)

**Comment by andygrillo** [comment]
Great ! Otherwise, it works brilliantly.
